### DISCUSSION TITLE
Automated Link/URL checker

### DISCUSSION THREAD

**Discussion by dhmlau** [loopbackio/loopback-next#7202] [Ideas]
## Suggestion
I'd like to propose to use GitHub Action to check whether links referenced in our docs exist. Possible criteria:

- check on each pull request on loopback-next - if the changes in PR causes any broken links
  - if it has broken link, mark CI as failed
- run a cron job (maybe in loopback.io?) - run daily or periodically to make sure we don't have broken links
   - if it has broken link, open a PR automatically
- able to specify an ignore list - we have quite a lot of references on `http://localhost:3000/explorer` or similar URL for the API explorer.
- (optional) able to specify the directory to be scanned - supposedly we only want to scan `/docs` folder

### Some investigation
I did some investigation and found https://github.com/marketplace/actions/link-checker might be the closest of what we want, but it doesn't have the "ignore list" capability. 

Note: I wasn't able to get it to show as error in the checks yet at the time of opening this issue, it might be because I don't have the right configuration. 
<img width="559" alt="Screen Shot 2020-08-26 at 9 00 31 PM" src="https://user-images.githubusercontent.com/25489897/91371545-46d04900-e7df-11ea-9c52-29dd75b7ba25.png">

**Comment by achrinza** [comment]
Created #6208 to demonstrate a working (?) example.
